src/components/SignUpForm.tsx

```diff
@@ -15,7 +15,12 @@ const SignUp: Component = () => {
       initialValues={{ email: '', password: '' }}
       validation={{
         email: Yup.string().email().required(),
-        password: Yup.string().required(),
+        password: Yup.string()
+          .required('Please enter your password')
+          .matches(
+            /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#\$%\^&\*])(?=.{6,})/,
```

Comment:
This is some magical looking RegEx! 🪄

Reply:
Ya i saw yup-password as well, but wasnt sure if you guys wanted another package. I can try that out! :)

Reply:
Sure, I'm all for using packages. In an ideal world, I'm a 10x developer that writes everything myself. In this one-product-a-mont-world, let's build on the shoulders of giants! 😅